src/com/lftechnology/java/training/alina/basic/guessnumber/NumberGuessingGame.java

```diff
@@ -5,58 +5,55 @@
 import java.util.logging.Logger;
 
 /**
- * Write a Guessing a number game application: application will generate a
- * random no between 1 and 20. user guess the value by entering from the console
- * Check if the guess value matched with random number or not, if matched show
- * the guessing value, random value and the no. of attempts.
+ * Write a Guessing a number game application: application will generate a random no between 1 and 20. user guess the value by entering from
+ * the console Check if the guess value matched with random number or not, if matched show the guessing value, random value and the no. of
+ * attempts.
  * 
  * @author Alina Shakya <alinashakya@lftechnology.com>
  *
  */
 public class NumberGuessingGame {
 
-	private static final Logger LOGGER = Logger.getLogger(NumberGuessingGame.class.getName());
+    private static final Logger LOGGER = Logger.getLogger(NumberGuessingGame.class.getName());
 
-	public static void main(String[] args) {
-		int randomNumber;
-		int maxLimit = 20;
-		int minLimit = 1;
-		try (Scanner scanner = new Scanner(System.in)) {
-			NumberGuessingGame number = new NumberGuessingGame();
-			randomNumber = Randomize.generateRandomNumber(minLimit,maxLimit);
-			number.checkForGuessNumber(scanner, randomNumber);
-		} catch (Exception e) {
-			LOGGER.log(Level.WARNING, "Exception Message : {0}", new Object[] {e.getMessage()});
-		}
-	}
+    public static void main(String[] args) {
+        int randomNumber;
+        int maxLimit = 20;
+        int minLimit = 1;
+        try (Scanner scanner = new Scanner(System.in)) {
+            NumberGuessingGame number = new NumberGuessingGame();
+            randomNumber = Randomize.generateRandomNumber(minLimit, maxLimit);
+            number.checkForGuessNumber(scanner, randomNumber);
+        } catch (Exception e) {
```

Comment:
![CRITICAL](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png) Either log or rethrow this exception. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1166)